synapse/cores/sqlite.py

```diff
@@ -337,6 +344,24 @@ def _initCoreStor(self):
 
         self._initCorTables(table)
 
+    def _postCoreRegistration(self, core):
+        # XXX This optimization for 'range' only exists for sqlite/psql
+        # We should move this into the storage layer and add
+        # ram / lmdb implementations if possible
+        self.initTufosBy('range', self._tufosByRange)
+        def rangefini():
```

Comment:
This was for dec-ref purposes, so ideally after fini'ing we'd have a ref from the Cortex to the storage object and that would be it (no circular refs in theory).   This has been unified into a single place.